includes/manage_pages.inc

```diff
@@ -731,14 +734,18 @@ function islandora_paged_content_zipped_upload_form_submit(array $form, array &$
     );
     $file_count = count($files_to_add);
     $config = array(
-      'language' => $language,
       'pid' => $pid,
       'destination_dir' => $destination_dir,
       'namespace' => $namespace,
       'language' => $form_state['values']['language'],
       'model' => $form_state['values']['model'],
       'file_count' => $file_count,
       'book_label' => $book_label,
+      'language' => isset($form_state['values']['language']) ? $form_state['values']['language'] : 'English',
```

Comment:
Not sure if an issue, but I think the value is actually "eng", as set in the previous default and not "English"? Or the other way around? but both need to be consistent.

Reply:
Yeah, whoops